gitpod-io/website
# feature: move education customers to for education

## Description

This PR
- Removes education customers from /customers, so that page can focus on enterprise
- Adds permanent redirect from `/customers/bruno-rocha` and `/customers/4geeks` to their `/for/education` counterparts so inbound links don't break
- Adds education customers to the `/for/education` landing page

## How to test

Navigate to `/customers`. The customer stories should look like this and not contain any education stories

<img width="1121" alt="image" src="https://user-images.githubusercontent.com/5633704/177975720-28adae37-6647-4ac2-aa0b-3eb4f7e7b4dd.png">

Navigate to `/customers/4geeks` by manually entering it in the URL bar. It should redirect you to `/for/education/4geeks`
Navigate to `/customers/bruno-rocha` by manually entering it in the URL bar. It should redirect you to `/for/education/bruno-rocha`

Navigate to `/for/education`. You should now see customer stories near the bottom of the page

<img width="1246" alt="image" src="https://user-images.githubusercontent.com/5633704/177974665-6e6fc3bf-ff81-4430-b347-a855871bb63c.png">



## Release Notes

```release-note
```

## Documentation
